### PR TITLE
Config test

### PR DESCRIPTION
**Pull request by dokun1**
I've updated the readme. My girlfriend told me I was "doing the writeme". Lol.

### CODE REVIEW

File: iOS/rainbow/Config/GameObjects.json [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
- "colorIcon": "Plant.png",
- "whiteIcon": "Plant---White.png"
- },
- {
- "name": "Shirt",
- "colorIcon": "Shirt.png",
- "whiteIcon": "Shirt---White.png"
- }
-]
+ [
```
Comment (stevemart) [inline iOS/rainbow/Config/GameObjects.json]: https://jsonlint.com/ please!

File: README.md [hunk 2]
```diff
@@ -58,29 +59,85 @@ Build the project by navigating to the `iOS` folder on your terminal, and typing
...
+<p align="center">
+	<img src="./images/cloudantChoice.png" width=300>
+</p>
+3. Go to your Cloudant service home page, and click the green  **Launch** button. Click the database icon on the left, and along the top, click "Create Database". Name it `routes-users`, and click "Create Document". Edit your JSON to include a `"username"` and a `"password"` of your choosing. If you do this, you will need this when you set up your iOS application.
+4. Do the same thing for a Push Notifications service as well.
+ <p align="center">
+	<img src="./images/pushNotificationsChoice.png" width=300>
+</p>
+ 
+ (**to set up push notifications with your app, you will need to follow the [guide](https://console.bluemix.net/docs/services/mobilepush/index.html#gettingstartedtemplate) for embedding them into your app**)
```
Comment (stevemart) [inline README.md]: use a `>` for a note instead

File: iOS/rainbow/Config/GameObjects.json (hunk 1 above)
Comment (dokun1) [reply iOS/rainbow/Config/GameObjects.json]: oh god im so sorry

File: README.md (hunk 2 above)
Comment (dokun1) [reply README.md]: Ok, let's add that after because I am struggling to see how that affects it

**Pull request merged**